libraries/src/main/java/com/baeldung/flink/LineSplitter.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.flink;
+
+import org.apache.flink.api.common.functions.FlatMapFunction;
+import org.apache.flink.api.java.tuple.Tuple2;
+import org.apache.flink.util.Collector;
+
+public class LineSplitter implements FlatMapFunction<String, Tuple2<String, Integer>> {
```

Comment:
add @SuppressWarnings("serial") to class or create the default serial version ID